docs/content/en/docs/contribution-guidelines/development.md

```diff
@@ -16,6 +16,11 @@ You can install `bazelisk` by `go get` as the following:
 go get github.com/bazelbuild/bazelisk
 ```
 
+If your Go version is 1.17 or higher, `go install` is recommended:
```

Comment:
Since the installation of bazelisk may be changed later, how about refer the installation note from the official docs here to avoid outdated docs?
https://github.com/bazelbuild/bazelisk#requirements

Reply:
I agree with your opinion. I'll update it.

Reply:
@khanhtc1202 

I updated it. Could you check it?